src/ec/packages/ec-component-footer-core/footer-core.story.js

```diff
@@ -1,14 +1,113 @@
+/* eslint-disable no-param-reassign, no-shadow */
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
+import { withKnobs, text } from '@storybook/addon-knobs';
 
-import sections from './demo/data';
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import specs from './demo/data';
 import footer from './ecl-footer-core.html.twig';
 import notes from './README.md';
 
+const data = [...specs.sections];
+// Labels for the buttons.
+const requiredGroupId = 'Mandatory elements';
+const optionalGroupId = 'Optional elements';
+// Prepare the knobs.
+const formatFooter = data => {
+  // Site name.
+  data[0].title.link = {
+    label: text(
+      'sections[0].title.link.label',
+      data[0].title.link.label,
+      requiredGroupId
+    ),
+    path: text(
+      'sections[0].title.link.path',
+      data[0].title.link.path,
+      requiredGroupId
+    ),
+    placeholder: text(
+      'optional elements',
+      'no optional elements defined in this story.',
+      optionalGroupId
+    ),
```

Comment:
I would like a better label for the knobs. even if we make the label for developers this should make more sense. Also i don't see the knob  for " the description"  underneath the footer title.

Reply:
and do you have any better proposal? As we know this is sub-optimal anyhow, we need to define it just in order to see the tabs (mandatory/optional) when we have no optional element, that's why it literally says what we want to "communicate", that there is no optional element. Open to alternative solutions but unless we don't have a better one i think we need it.